device/iot-device-samples/device-reconnection-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/SampleParameters.java

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.azure.core.http.policy.HttpLogDetailLevel;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+import java.net.URISyntaxException;
+import java.io.*;
+
+
+public class SampleParameters {
+
+    private static final String _DEVICE_CONNECTION_STRING = System.getenv("IOTHUB_DEVICE_CONNECTION_STRING");
+    private static final String _TRANSPORT = "mqtt";
+    private static final String _NUM_REQUESTS = "3";
+    private static final String _SLEEP_DURATION_IN_SECONDS = "10";
+    private static final String _TIMEOUT_IN_MINUTES = "1";
```

Comment:
We don't typically prefix our private static final fields with an underscore

Reply:
```java
private static final String TIMEOUT_IN_MINUTES = "1";
```
Would match what we normally do

Reply:
My old habit, will match.